source/Background/index.ts

```diff
@@ -239,7 +239,7 @@ async function handleMessage(
           zapclosewindowhandle: false,
         });
         if (zapclosewindowhandle) {
-          const stmt = new ZestStatementWindowClose(0);
+          const stmt = new ZestStatementWindowClose(0, 0);
```

Comment:
I've left the wait zero here. I'm not sure if that's the right choice or not.
Have a look through the full changes then let me know if you disagree.

Reply:
ZestStatementWindowClose doesnt support waitForMsec (see other comment).